widget/button.go

```diff
@@ -33,20 +33,27 @@ type Button struct {
 	textPadding              Insets
 	graphicPadding           Insets
 
-	init      *MultiOnce
-	widget    *Widget
-	container *Container
-	graphic   *Graphic
-	text      *Text
-	hovering  bool
-	pressing  bool
-	state     WidgetState
+	textInitOpts *textInitOptions
+	init         *MultiOnce
+	widget       *Widget
+	container    *Container
+	graphic      *Graphic
+	text         *Text
+	hovering     bool
+	pressing     bool
+	state        WidgetState
 
 	tabOrder      int
 	focused       bool
 	justSubmitted bool
 }
 
+type textInitOptions struct {
```

Comment:
The intention was to make the button object less "heavy" by only adding a single pointer to it that is nulled after the initialization, since we don't need it afterwards.
Not a big deal I guess, I can get rid of that struct.